code/datums/jobs/job/shipside.dm

```diff
@@ -14,7 +14,7 @@
 	supervisors = "TGMC high command"
 	selection_color = "#ccccff"
 	skills_type = /datum/skills/captain
-	access = ALL_MARINE_ACCESS
+	access = list(ALL_MARINE_ACCESS, ACCESS_MARINE_TANK)
```

Comment:
This won't work since ALL_MARINE_ACCESS is already a list, you're basically making this a 2 deep list